src/opacity/absorber_ck.hpp

```diff
@@ -33,7 +33,8 @@ class HeliosCKPremix: public AbsorberCK {
   virtual ~HeliosCKPremix() {}
 
   void LoadCoefficient(std::string fname, size_t bid) override;
+  void LoadCoefficient(std::string fname, size_t bid, std::ostream &os);
```

Comment:
I understand that this is for testing purposes to include "os". After you have tested it, you can remove it from the function API.